Source/ROLib/Modules/ROLAnimateEngineHeat.cs

```diff
@@ -124,13 +129,27 @@ private void locateAnimatedTransforms()
             if (animatedRenderers == null || animatedRenderers.Length == 0) { print("ERROR: Could not locate any emissive meshes for name: " + meshName); }
         }
 
+        private void locateSolverEngines()
+        {
+            ModuleEngines solverEnginesModule = ROLModInterop.getSolverEngineModule(part, engineID);
+
+            if (solverEnginesModule == null)
+                return;
```

Comment:
There's a neat construct in C# with the "is" operator that fits this pattern.

```if (ROLModInterop.getSolverEngineModule(part, engineID) is ModuleEngines solverEnginesModule)
{
    // rest of code
}```
Instead of the declaration, assignment, and then null check

Reply:
Yeah, that is definetely nicer, I even used it in the new ROLModInterop methods but I always forget about these nice things C# provides